cluster/gce/config-default.sh

```diff
@@ -74,6 +74,14 @@ DNS_SERVER_IP="10.0.0.10"
 DNS_DOMAIN="cluster.local"
 DNS_REPLICAS=1
 
+# Optional: Create autoscaler for cluster's nodes.
+ENABLE_NODE_AUTOSCALER="${ENABLE_NODE_AUTOSCALER:-false}"
```

Comment:
Please make the name of env vars consistent with other (instead of 2nd occurence ENABLE_NODE_AUTOSCALER use KUBE_ENABLE_NODE_AUTOSCALER).

The same below